sdks/smart-wallet-sdk/src/constants.ts

```diff
@@ -42,78 +39,62 @@ export enum SupportedChainIds {
  */
 export enum SmartWalletVersion {
   LATEST = 'latest',
-  v0_3_0_audit_2 = 'v0.3.0-audit.2',
-  v0_2_1_audit_2 = 'v0.2.1-audit.2',
+  v1_0_0_staging = 'v1.0.0-staging',
+  v0_3_0_audit_2 = 'v0.3.0-audit.2'
 }
 
+// All smart wallet versions for a given chain id are optional except for the latest version
+type SmartWalletVersionMap = Partial<{ [version in SmartWalletVersion]: `0x${string}` }> & { [SmartWalletVersion.LATEST]: `0x${string}` }
+
 /**
  * Smart wallet versions for supported chains
  */
-export const SMART_WALLET_VERSIONS: { [chainId in SupportedChainIds]: { [version in SmartWalletVersion]: `0x${string}` } } = {
+export const SMART_WALLET_VERSIONS: { [chainId in SupportedChainIds]: SmartWalletVersionMap } = {
   [SupportedChainIds.MAINNET]: {
-    [SmartWalletVersion.LATEST]: '0x458f5a9f47A01beA5d7A32662660559D9eD3312c',
+    [SmartWalletVersion.LATEST]: '0x3cbad1e3b9049ecdb9588fb48dd61d80faf41bd5',
+    [SmartWalletVersion.v1_0_0_staging]: '0x3cbad1e3b9049ecdb9588fb48dd61d80faf41bd5',
     [SmartWalletVersion.v0_3_0_audit_2]: '0x458f5a9f47A01beA5d7A32662660559D9eD3312c',
```

Comment:
whats the reason to keep these old versions?

Reply:
The frontend wants to be able to test flows where the user is migrating between versions